src/routes/+page.svelte

```diff
@@ -543,6 +543,7 @@
                                 use:tippy={{
                                     arrow: true,
                                     allowHTML: true,
+                                    interactive: true,
```

Comment:
Yea right, but I meant add dropdown ( tippy ) for all buttons with arrow icon
like add select dropdown for this:
![Screenshot from 2023-06-09 18-25-47](https://github.com/tokitou-san/CoreProject-V3-UI/assets/114811070/44fcc901-90c8-46c7-94bb-9199ef765cb8)

If you're okay, we can do this thing later?

Reply:
> If you're okay, we can do this thing later?

 Sure